README.md

```diff
@@ -83,7 +89,27 @@ See [customize options] and [manage connection profiles] for more details.
 ```
 
 ## Change Log
-The current version is ```0.2.0```. See the [change log] for more detail.
+The current version is ```0.2.0```. See the [change log] for a detailed list of changes in each version.
+
+## Supported Operating Systems
+
+Currently this extension supports the following operatings systems:
+
+* Windows (64-bit only)
+* macOS
```

Comment:
10.11 and by the way we might have a bug that we don't say 10.12 is not supported. we have to verify

---
In reply to: [92522143](https://github.com/Microsoft/vscode-mssql/pull/531#discussion_r92522143) [](ancestors = 92522143)